pandas/core/indexes/datetimes.py

```diff
@@ -173,6 +179,45 @@ class DatetimeIndex(DatetimeArrayMixin, DatelikeOps, TimelikeOps,
     TimedeltaIndex : Index of timedelta64 data
     PeriodIndex : Index of Period data
     pandas.to_datetime : Convert argument to datetime
+
+    Examples
+    --------
+
+        Localize local times:
+
+        >>> s = pd.to_datetime(pd.Series(['2018-09-15 01:30:00']))
+        >>> pd.DatetimeIndex(s, tz='CET')
```

Comment:
Would be better to show `tz_localize` once this docstring is under the `tz_localize` method docstring.